src/cpp/session/modules/connections/SessionConnections.cpp

```diff
@@ -74,7 +76,7 @@ SEXP rs_connectionOpened(SEXP connectionSEXP)
    if (!r::sexp::isNull(actionList))
    {
       std::vector<std::string> actionNames;
-      error = r::sexp::getNames(actionList, &actionNames);
+      r::sexp::getNames(actionList, &actionNames);
```

Comment:
Gotcha -- I think the following `if (error) LOG_ERROR(error);` bit could be stripped out then; does that sound right?